Packs/FeedAutofocus/Integrations/FeedAutofocus/README.md

```diff
@@ -0,0 +1,71 @@
+Use the AutoFocus Feeds integration to fetch indicators from AutoFocus.
+For more information see the [AutoFocus documentation](https://docs.paloaltonetworks.com/autofocus/autofocus-admin/autofocus-feeds.html).
+
+## Configure AutoFocus Feed on Demisto
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for AutoFocus Feed.
+3. Click **Add instance** to create and configure a new integration instance.
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| feed | Fetch indicators | False |
+| indicator_feeds | Indicator Feed - Choose the requested indicator feeds - Custom Feeds and Daily Threat Feed. | True |
+| api_key | The AutoFocus API key | True |
+| custom_feed_urls | The URL for the custom feed to fetch - Only in cases a Custom Feed is requested | False |
+| feedReputation | Indicator Reputation | False |
+| feedReliability | Source Reliability | True |
+| feedExpirationPolicy |  | False |
+| feedExpirationInterval |  | False |
+| feedFetchInterval | Feed Fetch Interval | False |
+| feedBypassExclusionList | Bypass exclusion list | False |
+| insecure | Trust any certificate \(not secure\) | False |
+| proxy | Use system proxy settings | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+
+#### Custom Feed info:
+To connect a custom AutoFocus feed you need to provide the Custom Feed URL.
+
+The Custom Feed URL should be in this form:
+https://autofocus.paloaltonetworks.com/IOCFeed/{Output_Feed_ID}/{Output_Feed_Name}
+
+
+## Commands
+You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### autofocus-get-indicators
+***
+Gets the indicators from AutoFocus.
+Note: This command does not create indicators within Cortex XSOAR.
+
+##### Base Command
+
+`autofocus-get-indicators`
+##### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| limit | The maximum number of indicators to return. The default value is 10. | Optional | 
```

Comment:
no need to add `The default value is...`, Demisto does it automatically

Reply:
@roysagi lets fix this one and merge

Reply:
I'll leave it in the readme and drop it in the yml as it does need to appear in the documentation